README.md

```diff
@@ -1,42 +1,53 @@
 [![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) ![dbt logo and version](https://img.shields.io/static/v1?logo=dbt&label=dbt-version&message=1.x&color=orange)
 
-# Medicare Claims Connector
+# Medicare CCLF Claims Connector
 
-Check out the Tuva Project [Docs](http://thetuvaproject.com/)
+## 🧰 What does this project do?
 
-Check out the Tuva Project [Data Models](https://docs.google.com/spreadsheets/d/1NuMEhcx6D6MSyZEQ6yk0LWU0HLvaeVma8S-5zhOnbcE/edit?usp=sharing)
+This connector transforms raw Medicare CCLF claims data into the Tuva Claims Input Layer which enables you to run most of the other components of the Tuva Project with very little effort.
+For a detailed overview of what the project does and how it works, check out our [Knowledge Base](https://thetuvaproject.com/docs/getting-started). 
+For information on data models and to view the entire DAG check out our dbt [Docs](https://tuva-health.github.io/medicare_cclf_connector/#!/overview?g_v=1).
 
-Check out the [DAG](https://tuva-health.github.io/medicare_cclf_connector/#!/overview?g_v=1) for this dbt project
+## 🔌 Database Support
 
-## Description
-This connector transforms raw Medicare CCLF claims data into the Tuva Claims Input Layer which enables you to run most of the other components of the Tuva Project with very little effort.
+- BigQuery
+- Redshift
+- Snowflake
+
+## ✅ How to get started
```

Comment:
Kept the original steps here since this is not a package like the rest of the Tuva Project.